packages/vuepress-theme-odyssey/components/DocsPageHeader.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <header
+    :class="{
+      'docs-page-header': true,
+      [`is-page-header-${variant}`]: true
+    }"
+  >
+    <h1 class="docs-page-header--title">{{ title }}</h1>
+    <p v-if="lead" class="docs-page-header--lead">{{ lead }}</p>
+    <slot name="right"></slot>
```

Comment:
We shouldn't use directional names - prep for internationalization

Reply:
Good feedback! Any ideas on what to call this?

Reply:
Moved conversation to #728